dvc/api/__init__.py

```diff
@@ -139,4 +139,4 @@ def make_checkpoint():
         fobj.flush()
         os.fsync(fobj.fileno())
     while os.path.exists(signal_file):
-        sleep(1)
+        sleep(0.1)
```

Comment:
we could move this into a constant and patch it only for tests. Kinda like we do with moto https://github.com/iterative/dvc/blob/08300d3bcf51f7bd9c8509924110416461b11b60/tests/func/test_s3.py#L13

Reply:
Dmitry has mentioned before that he thought this 1s sleep could be sped up in general, so this change doesn't really need to be for tests only.

Reply:
Still better to not hardcode such things :slightly_smiling_face: